e2e/tests/test_cases/physical_node_reboot.robot

```diff
@@ -0,0 +1,52 @@
+*** Settings ***
+Documentation    Physical node reboot
+...              https://longhorn.github.io/longhorn-tests/manual/pre-release/node-not-ready/node-restart/physical-node-reboot/
+
+Test Tags    manual_test_case
+
+Resource    ../keywords/common.resource
+Resource    ../keywords/deployment.resource
+Resource    ../keywords/persistentvolumeclaim.resource
+Resource    ../keywords/statefulset.resource
+Resource    ../keywords/volume.resource
+Resource    ../keywords/workload.resource
+Resource    ../keywords/host.resource
+
+Test Setup    Set test environment
+Test Teardown    Cleanup test resources
+
+*** Variables ***
+${LOOP_COUNT}    1
+${RETRY_COUNT}    300
+${RETRY_INTERVAL}    1
+
+*** Test Cases ***
+Physical Node Reboot With Attached Deployment
+    Given Create persistentvolumeclaim 0 using RWO volume
+    And Create persistentvolumeclaim 1 using RWX volume
+    And Create deployment 0 with persistentvolumeclaim 0
+    And Create deployment 1 with persistentvolumeclaim 1
+    And Write 100 MB data to file data in deployment 0
+    And Write 100 MB data to file data in deployment 1
+    And Reboot volume node of deployment 0
+    And Reboot volume node of deployment 1
+
+    And Wait for deployment 0 pods stable
+    Then Check deployment 0 data in file data is intact
+
+    And Wait for deployment 1 pods stable
```

Comment:
`Wait for workloads pods stable` can be used to save wait time.

Reply:
Updated!